charmcraft.yaml

```
@@ -61,7 +61,10 @@ actions:
       Regenerate the world state from scratch. This includes the configuration file,
       but also relation data set by the charm.
 
-# provides:
+provides:
+  logging-provider:
```

If we can use a relation name different from gagent, then perhaps:
```suggestion
  receive-loki-logs:
```
Wdyt @simskij ?